ols/app/endpoints/ols.py

```diff
@@ -310,7 +313,7 @@ def store_conversation_history(
             status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
             detail={
                 "response": "Error storing conversation",
-                "cause": str(e),
+                "cause": f"{type(e).__name__}: {e}",
```

Comment:
it's repeated, prob. make a helper function then (when somebody ask to change format, it should be made on one place)